lib/src/translations.dart

```diff
@@ -20,11 +32,12 @@ class Translations {
       if (value is Map<String, dynamic>) value = value[keys[i]];
     }
 
-    if (value == null) {
-      throw Exception('Cannot cache a key that is not nested.');
```

Comment:
The problem with this throw is it's an uncaught exception so it crashes the entire app.

Reply:
yeah agreed with u  :100: